R/api.R

```diff
@@ -446,3 +447,45 @@ endpoint_workflow_status <- function(runner) {
 check_timeout <- function(runner, rate_limit = 2 * 60) {
   throttle(runner$check_timeout, rate_limit)
 }
+
+
+target_report_version_artefact <- function(path, id) {
+  db <- orderly::orderly_db("destination", root = path)
+  sql <- paste(
+    "select",
+    "       report_version_artefact.'order' as id,",
```

Comment:
I can easily flip this over to use `id` as it currently is, but I think you really want `order` here (which I can rename to `id` so you don't have to change things).

The `id` column is a `SERIAL` over the `report_version_artefact` table whereas the `order` column is a set of integers 1, 2, ... that orders artefacts within a report (so `id` is unique over all reports but `order` is not, but they have the same ordering within a report due to some business logic in orderly).

It might be worth quickly seeing what you are doing with this column before changing? The `id` should ideally not make it out as it could change between runs.  If all you are doing is a sort/order based on this, or if you're ignoring it, this change should not matter

Reply:
I guess data types vary then because in SQLlite order and id are integer data types. I don't think the id is important anymore as you have actually joined the table  on file_artefact.artefact = report_version_artefact.id", which is essentially why the id was selected from report_version_artefact in SQLLite.